recipes/boost/all/dependencies/dependencies-1.69.0.yml

```diff
@@ -0,0 +1,267 @@
+configure_options:
+- chrono
+- container
+- contract
+- coroutine
+- date_time
+- exception
+- fiber
+- filesystem
+- graph
```

Comment:
I think we are missing a couple of options from the configuration list. (I think) the full list is visible during the build: 
```
    - atomic                   : building
    - chrono                   : building
    - container                : building
    - context                  : building
    - contract                 : building
    - coroutine                : building
    - date_time                : building
    - exception                : building
    - fiber                    : building
    - filesystem               : building
    - graph                    : building
    - graph_parallel           : building
    - headers                  : building
    - iostreams                : building
    - locale                   : building
    - log                      : building
    - math                     : building
    - mpi                      : not building
    - nowide                   : building
    - program_options          : building
    - python                   : not building
    - random                   : building
    - regex                    : building
    - serialization            : building
    - stacktrace               : building
    - system                   : building
    - test                     : building
    - thread                   : building
    - timer                    : building
    - type_erasure             : building
    - wave                     : building
```

Reply:
@madebr do we still have missing things here?

Reply:
I think everything is there.
I've tested with python, but not touched mpi at all.
When my cpython pr lands, then that option would be completely functional (it is now, but system dependent)

Maybe adding an openmpi recipe would make the openmpi option available, but that's work for future pr's.

Reply:
okay, MPI is big thing on its own